src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExceptionListenerPass.php

```diff
@@ -36,5 +36,9 @@ public function process(ContainerBuilder $container)
                 $container->removeDefinition('twig.exception_listener');
             }
         }
+
+        if ($container->hasDefinition('twig.exception_listener')) {
```

Comment:
@dmaicher What do you think of putting the responsibility in the existing TwigBundle compiler pass?